### PR TITLE
Objective-C interoperability quick start one

### PR DESCRIPTION
**Pull request by ericgiannini**

### CODE REVIEW

**Comment by ericgiannini** [comment]
@shams-ahmed There is an ambiguity in the codebase w.r.t. the conversion of `.subscribe` into an Objective-C equivalent.

**Comment by ericgiannini** [comment]
@shams-ahmed Please let me know as soon as you put in a patch for the correct exposure.

File: docs/1-simple-conversation-objc.md [hunk 1]
```diff
@@ -0,0 +1,282 @@
...
+@import Stitch;
+
+@interface LoginVC : UIViewController
+
+@property (retain, nonatomic) NXMConversationClient *client;
+
+@end
+```
+
+Program the implementation file in the following way:
```
Comment (AlexLakatos) [inline docs/1-simple-conversation-objc.md:69]: why is this italic?

File: docs/1-simple-conversation-objc.md [hunk 2]
```diff
@@ -0,0 +1,282 @@
...
+    [super viewDidLoad];
+    // Do any additional setup after loading the view.
+    
+    client = [NXMConversationClient instance];
+}
+
+@end
+```
+
+### 2.4 Creating the login layout
```
Comment (AlexLakatos) [inline docs/1-simple-conversation-objc.md:99]: seems there is some sort of parsing error after the first code sample, can you look into that?

File: docs/1-simple-conversation-objc.md [hunk 3]
```diff
@@ -0,0 +1,282 @@
...
+Configure your own layout for login with: 
+
+1. an instance of UIButton called `loginBtn`
+2. an instance of UIButton called `chatBtn`
+3. an instance of UILabel called `statusLbl`
+4. an instance of UINavigationController embedded in the scene for what is assigned to the LoginVC. 
+
+### 2.5 - Create the Login Functionality
+
+Below `UIKit` let's import the `NexmoConversation`. Next we setup a custom instance of the `ConversationClient` and saving it as a member variable in the view controller.
```
Comment (AlexLakatos) [inline docs/1-simple-conversation-objc.md]: isn't this renamed to `NexmoStitch`?

File: docs/1-simple-conversation-objc.md [hunk 4]
```diff
@@ -0,0 +1,282 @@
...
+
+```Objective-C 
+NXMEventCollection *events = [conversation events];
+    
+NXMEventBase *baseEvent = [events indexWithSafe:[events startIndex]];
+    
+[baseEvent isCurrentlyBeingSent];
+```
+
+## 3.0 - Trying it out
```
Comment (AlexLakatos) [inline docs/1-simple-conversation-objc.md:280]: Please add a "What's next" section, same as Android and JS have

File: docs/1-simple-conversation-objc.md [hunk 5]
```diff
@@ -0,0 +1,282 @@
...
+
+After this you should be able to run the app and send messages.
```
Comment (AlexLakatos) [inline docs/1-simple-conversation-objc.md]: add a link to the code example folder on GitHub please

**Review by AlexLakatos — CHANGES_REQUESTED** [review]

File: docs/1-simple-conversation-objc.md (hunk 1 above)
Comment (ericgiannini) [reply docs/1-simple-conversation-objc.md:69]: It shouldn't be.

File: docs/1-simple-conversation-objc.md (hunk 2 above)
Comment (ericgiannini) [reply docs/1-simple-conversation-objc.md:99]: I didn't see in MacDown.

File: docs/1-simple-conversation-objc.md (hunk 3 above)
Comment (ericgiannini) [reply docs/1-simple-conversation-objc.md]: `Stitch`

File: docs/1-simple-conversation-objc.md (hunk 4 above)
Comment (ericgiannini) [reply docs/1-simple-conversation-objc.md:280]: Added.

File: docs/1-simple-conversation-objc.md (hunk 5 above)
Comment (ericgiannini) [reply docs/1-simple-conversation-objc.md]: Added.

**Pull request merged**